library/Faktory/Job.hs

```diff
@@ -39,6 +42,7 @@ data Job arg = Job
   -- ^ Faktory needs to serialize args as a list, but we like a single-argument
   -- interface so that's what we expose. See @'jobArg'@.
   , jobOptions :: JobOptions
+  , jobFailure :: Maybe JobFailure
```

Comment:
What happens if you send a non-`Nothing` value here when queueing a job? Presumably it's just ignored?

Reply:
Presumably. I could put a temporary spec in to see, but we don't expose any ability for a caller of our code to do that, so I'm inclined to not worry.

Reply:
Yeah, was just curious if you knew off the top of your head